src/lwc/utilTemplateBuilder/utilTemplateBuilder.js

```diff
@@ -300,6 +300,12 @@ const dispatch = (context, name, detail, bubbles = false, composed = false) => {
 * @param {object} error: Event holding error details
 */
 const handleError = (error) => {
```

Comment:
More of a note for the future, but I'm not a huge fan of this particular `handleError` being imported outside of template builder use-cases. Ideally, we'd either move `handleError` out of `utilTemplateBuilder` and into `utilCommon` or start a new method entirely for general use.

Reply:
@psadie-sfdo I added a new method called `buildErrorMessage` into that component where we can get back the error message string without having to throw a toast but without disrupting all the other references to `handleError`, but I do agree the more we can pull utilities out of the `utilTemplateBuilder` the better but that would be a pretty high LOE at this point since we're importing this in a bunch of places.

Reply:
+++ on that update and completely agree that we shouldn't tackle pulling `handleError` out in this work.

Thanks!!